talend-component-maven-plugin/src/main/java/org/talend/sdk/component/maven/ValidateComponentMojo.java

```diff
@@ -190,14 +203,19 @@ public void doExecute() {
         configuration.setValidateNoFinalOption(validateNoFinalOption);
         configuration.setValidateWording(validateWording);
         configuration.setPluginId(pluginId);
+        configuration.setValidateExceptions(validateExceptions);
+        configuration.setFailOnValidateExceptions(failOnValidateExceptions);
 
         final Locale locale = this.locale == null || "root".equals(this.locale) ? Locale.ROOT : new Locale(this.locale);
-        new ComponentValidator(configuration, new File[] { classes }, getLog()) {
+        List<String> compileSourceRoots = project.getCompileSourceRoots();
+        new ComponentValidator(configuration, new File[] { classes }, compileSourceRoots, getLog()) {
```

Comment:
I don't see such constructor in ComponentValidator class?!?!

Reply:
Missed a commit here.